src/cart/index.html

```diff
@@ -80,31 +80,31 @@ <h2>My Cart</h2>
         <ul class="product-list">
           <li class="cart-card divider">
             <a
-              href="../product_pages/marmot-ajax-3.html"
+              href="/product_pages/marmot-ajax-3.html"
               class="cart-card__image"
             >
               <img
-                src="/src/public/images/tents/marmot-ajax-tent-3-person-3-season-in-pale-pumpkin-terracotta~p~880rr_01~320.jpg"
+                src="/images/tents/marmot-ajax-tent-3-person-3-season-in-pale-pumpkin-terracotta~p~880rr_01~320.jpg"
                 alt="Marmot Ajax tent"
               />
             </a>
-            <a href="/src/product_pages/marmot-ajax-3.html">
+            <a href="/product_pages/marmot-ajax-3.html">
               <h2 class="card__name">Marmot Ajax Tent - 3-Person, 3-Season</h2>
             </a>
             <p class="cart-card__color">Pale Pumpkin/Terracotta</p>
             <p class="cart-card__quantity">qty: 1</p>
             <p class="cart-card__price">$199.99</p>
           </li>
           <li class="cart-card divider">
-             <a href="/src/product_pages/marmot-ajax-3.html" 
+            <a href="/product_pages/marmot-ajax-3.html"
               class="cart-card__image"
             >
               <img
-                src="/src/public/images/tents/the-north-face-talus-tent-4-person-3-season-in-golden-oak-saffron-yellow~p~985rf_01~320.jpg"
+                src="/images/tents/the-north-face-talus-tent-4-person-3-season-in-golden-oak-saffron-yellow~p~985rf_01~320.jpg"
                 alt="Talus Tent - 4-Person, 3-Season"
               />
             </a>
-            <a href="/src/product_pages/marmot-ajax-3.html" > 
+            <a href="/product_pages/marmot-ajax-3.html">
```

Comment:
Please read over the following: [A Couple of Things](https://teams.microsoft.com/l/message/19:ONvd-meSzdPhQ7qoHbNOvZe-RN_MpiTD_jyg_ie-bJg1@thread.tacv2/1706234846233?tenantId=e6ac1d1f-d695-4ef1-91d4-94cddef8be11&groupId=c92e44d0-a4f2-495f-839d-b74cbffad343&parentMessageId=1706234846233&teamName=Web%20Frontend%20Development%20II&channelName=General&createdTime=1706234846233)

This is wrong. It will not work. It tells me you are running the app incorrectly. Do not use VSCode Live Server. This is not a simple web site. This is a web application. You MUST use `npm start` to run this code. See MS Teams for more information.